src/audio/Kconfig

```diff
@@ -330,6 +330,20 @@ config COMP_CODEC_ADAPTER
 
 rsource "codec_adapter/Kconfig"
 
+config COMP_AEC
+	bool "AEC component"
+	default y
+	help
```

Comment:
Maybe put this `n` as default and enable it only when necessary?

Reply:
Yep, makes sense. Default Y shows well the various build problems.